lib/visitor.rb

```diff
@@ -0,0 +1,26 @@
+class Visitor
+    attr_reader :name, :height, :spending_money, :preferences
+
+    def initialize(name, height, spending_money)
+        @name = name
+        @height = height
+        @spending_money = spending_money.delete("$").to_i
+        @preferences = []
+    end
+
+    def add_preference(preference)
+        preferences << preference
+    end
+
+    def tall_enough?(requirement)
+        if @height >= requirement
+            return true
+        else
+            return false
+        end
```

Comment:
Yes, this works, however if we have an if/else stmt that is only returning true or false, then usually we don't need the if/else and we can just write `@height >= requirement` since that statement will return a true or false.